redisinsight/api/src/common/models/session.ts

```diff
@@ -9,21 +10,31 @@ export interface ISessionMetadata {
 }
 
 export class SessionMetadata implements ISessionMetadata {
+  @Expose()
   @IsNotEmpty()
   @IsString()
   userId: string;
 
+  @Expose()
   @IsObject()
   data?: Record<string, any> = {};
 
+  @Expose({ groups: ['security'] })
```

Comment:
Nit: this should become `Groups.Security` constant as it's on a lot of places

Reply:
Correct but it is not a single place. Didn't want to introduce more changes in bunch of not related to this PR files